newton/tests/test_environment_group_collision.py

```
@@ -94,10 +94,11 @@ def test_particle_shape_collision_filtering(self):
 
         model = builder.finalize(device=self.device)
         state = model.state()
-        contacts = model.contacts()
+        collision_pipeline = newton.CollisionPipeline(model)
```

Ruff (0.15.20)

[error] Undefined name `newton`

(F821)